README.md

```diff
@@ -255,6 +255,14 @@ juju config docker-registry \
 
 >Note: If any of the swift config options are set, they must all be set.
```

Comment:
This won't be true anymore since `domain` isn't required.  Let's say something like "... if any of **the above** are set, they must all be set.  **Optional** params are noted below."  Or otherwise enumerate exactly which config opts are required vs optional.